src/main/java/de/tum/in/www1/artemis/service/ProgrammingSubmissionService.java

```diff
@@ -657,10 +657,10 @@ public ProgrammingSubmission findByIdWithEagerResultAndFeedback(long submissionI
      * @param submissionId the id of the programming submission
      * @return the locked programming submission
      */
-    public ProgrammingSubmission lockAndGetProgrammingSubmission(Long submissionId) {
+    public ProgrammingSubmission lockAndGetProgrammingSubmission(Long submissionId, int correctionRound) {
```

Comment:
The documentation is missing for the new parameter.

Reply:
I added it :)